packages/wonder-blocks-tooltip/src/components/__tests__/tooltip-anchor.test.js

```diff
@@ -474,41 +448,42 @@ describe("TooltipAnchor", () => {
             expect(activeState).toBe(true);
 
             // Act
-            ref?.dispatchEvent(new FocusEvent("focusout"));
-            wrapper?.instance().activeStateStolen();
+            // Blur the anchor
+            userEvent.tab();
 
             // Assert
             expect(mockTracker.giveup).not.toHaveBeenCalled();
         });
 
-        test("if hovered, remains active", async () => {
+        test("if hovered, remains active", () => {
             // Arrange
             const timeoutSpy = jest.spyOn(global, "setTimeout");
             let activeState = false;
-            const ref = await new Promise((resolve) => {
-                const nodes = (
-                    <TooltipAnchor
-                        anchorRef={resolve}
-                        onActiveChanged={(active) => {
-                            activeState = active;
-                        }}
-                    >
-                        Anchor Text
-                    </TooltipAnchor>
-                );
-                mount(nodes);
-            });
-            ref?.dispatchEvent(new FocusEvent("focusin"));
+            const nodes = (
+                <TooltipAnchor
+                    anchorRef={jest.fn()}
+                    onActiveChanged={(active) => {
+                        activeState = active;
+                    }}
+                >
+                    Anchor Text
+                </TooltipAnchor>
+            );
+            render(nodes);
```

Comment:
**style nit (non-blocking):** we could pass the JSX directly to `render()` instead of using `nodes`.  This is what we do in most of the other test cases.